package.json

```diff
@@ -1,32 +1,31 @@
 {
   "name": "@influential/elm-graphql",
-  "version": "1.8.0",
+  "version": "1.8.1",
   "description": "Elm code generator for GraphQL",
   "main": "lib/main.js",
   "bin": "bin/elm-graphql",
   "scripts": {
     "prepare": "tsc-transpile-only",
-    "tsc": "tsc-transpile-only",
```

Comment:
Was this causing issues?

Reply:
No, but it did prevent running the typescript compiler to see the (unfortunate number) of errors.